build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -129,6 +129,7 @@ steps:
           }
         ]
       SessionTimeout: 20
+    condition: and(succeeded(), eq(variables['signed'], true))
```

Comment:
could you just add this to your over pr?

Reply:
I'm holding off until I get a normal build because I'd have to keep manually restarting the build with signed set to true. If we merge this it'll cause a merge conflict there and will remind me to fix it.

Reply:
as long as this goes in before your other pr.

Reply:
Yeah that one is going to take a while to confirm it works. Thanks Apple.